lib/routemap_cli.c

```diff
@@ -1358,6 +1358,11 @@ void route_map_action_show(struct vty *vty, const struct lyd_node *dnode,
 			yang_dnode_get_string(
 				dnode,
 				"./rmap-set-action/frr-bgp-route-map:l3vpn-nexthop-encapsulation"));
+	} else if (IS_SET_ZEBRA_KERNEL_BYPASS(action)) {
+		vty_out(vty, " set route kernel-bypass %s\n",
```

Comment:
`set route kernel-bypass`, but in documentation we have `set kernel-bypass`. Which one is correct?